pyccel/codegen/pipeline.py

```diff
@@ -240,7 +240,7 @@ def get_module_dependencies(parser, mods=(), folders=()):
         # TODO: stop at object files, do not compile executable
         #       This allows for properly linking program to modules
         #
-        if not (language == "c" and codegen.is_module):
+        if language == "fortran":
```

Comment:
This will stop c programs compiling

Reply:
yes, We talk about this. I just forget to restore it back.